### DISCUSSION TITLE
Settings of video with burnded-in subtitle

### DISCUSSION THREAD

**Discussion by Nespula95** [SubtitleEdit/subtitleedit#6504] [Q&A]
Hi, I have a question. Is it possible to set the font when burning subtitles so that it is the same every time?
The thing is that I generate videos very often and every time I have to change the font and its size.
"Segoe Ul" and size 23 is predefined, and I still use "Verdana" 35.
If this can be changed somewhere, please let me know.
![image](https://user-images.githubusercontent.com/120167992/206654715-3061fd21-d2a9-43e8-ba0f-28f1f00b7306.png)

**Comment by niksedk** [comment]
OK, tried to make SE remember font in burn-in in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.10/SubtitleEditBeta.zip

Better?

**Comment by Nespula95** [comment]
Everything works.. Thanks for your help.
